loops-arrays-objects.js

```diff
@@ -74,18 +99,22 @@ const NOVIEmployees = [
 //   { firstName: 'Mark', lastName: 'Rensen', email: 'Mark.Rensen@novi.nl' },
 //  ];
 // ==========================================
+for (let i = 0; i < NOVIEmployees.length; i++) {
 
 
+    let b = NOVIEmployees[i]["firstName"] + "." + NOVIEmployees[i]["lastName"] + "@novi.nl";
+    let c = b;
+    NOVIEmployees[i]["email"] = c.toLowerCase();
```

Comment:
Gebruik van toLowerCase() is een slimme oplossing om ervoor te zorgen dat alle e-mailadressen in kleine letters worden weergegeven.